ci/jenkins/test.sh

```diff
@@ -512,6 +517,8 @@ function run_e2e {
     go mod edit -replace github.com/moby/spdystream=github.com/antoninbas/spdystream@v0.2.1 && go mod tidy
     if [[ $FLEXIBLE_IPAM == true ]]; then
         go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus --antrea-ipam
+    elif [[ $TESTCASE =~ "multicast" ]]; then
+        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus --run=TestMulticast
```

Comment:
Is it a must here to specify test cases "run=TestMulticast"?

Reply:
Updated.